### PR TITLE
Add the Option to configure husky for jest

### PR DESCRIPTION
**Pull request by Strulle**
If wanted, you can add a git pre-push hook with husky to test your application with jest.

### CODE REVIEW

File: packages/schematics/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ and configured automatically for you.
 | ----------------------------------- | ------------------------------------------------------------ |
 | `ng g @co-it/schematics:commitlint` | Configure [commitlint] and set up commit hook using [husky]. |
 | `ng g @co-it/schematics:cypress`    | _coming soon_                                                |
-| `ng g @co-it/schematics:jest`       | _coming soon_                                                |
+| `ng g @co-it/schematics:jest`       | Configure [jest] and set up pre-push hook using [husky].                                                |
```
Comment (GregOnNet) [inline packages/schematics/README.md]: **Rephrase Proposal**
Set up [jest] and optionally configure a pre-push hook using [husky].

File: packages/schematics/README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ and configured automatically for you.
...
 | `ng g @co-it/schematics:prettier`   | Configure [prettier] and set up commit hook using [husky].   |
```
Comment (GregOnNet) [inline packages/schematics/README.md]: **Rephrase Proposal**
Set up [prettier] and optionally configure a commit hook using [husky].

File: packages/schematics/README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ and configured automatically for you.
 | ----------------------------------- | ------------------------------------------------------------ |
 | `ng g @co-it/schematics:commitlint` | Configure [commitlint] and set up commit hook using [husky]. |
 | `ng g @co-it/schematics:cypress`    | _coming soon_                                                |
```
Comment (GregOnNet) [inline packages/schematics/README.md]: **Add Documentation**
Set up cypress using [@nrwl/builders]

[@nrwl/builders](https://www.npmjs.com/package/@nrwl/builders)

File: packages/schematics/README.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ and configured automatically for you.
 | ----------------------------------- | ------------------------------------------------------------ |
 | `ng g @co-it/schematics:commitlint` | Configure [commitlint] and set up commit hook using [husky]. |
```
Comment (GregOnNet) [inline packages/schematics/README.md]: **Rephrase Proposal**
Set up [commitlint] and optionally configure a commit hook using [husky].

File: packages/schematics/src/jest/templates/hook/.huskyrc [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-push": "jest --watch=false"
```
Comment (GregOnNet) [inline packages/schematics/src/jest/templates/hook/.huskyrc]: The default is that Jest runs without watch mode

```
"pre-push": "jest"
```

**Comment by Strulle** [comment]
> Update
> 
> * Readme
> * hook template

@GregOnNet: done 🙂

**Pull request merged**